pkg/cli/cli_test.go

```diff
@@ -120,17 +140,17 @@ var _ = Describe("build cli", func() {
 			consoleLogErr:  "",
 		}, {
 			description:    "should validate with omission error",
-			args:           "validate-operating-parameters FALSE 1234 gcr.io/solo-public-1010",
+			args:           "validate-operating-parameters FALSE 1234 gcr.io/solo-public-1010 ",
```

Comment:
looks like cobra gets arguments by splitting on `" "`, could you remove the trailing space, then restore line 148?

Reply:
Sure!